nncf/quantization/fake_quantize.py

```diff
@@ -121,7 +122,11 @@ def tune_range(
         qval = fns.round(fval)
 
     ra = fns.where(qval < level_high, qval / (qval - level_high) * right_border, left_border)
-    rb = fns.where(qval > 0.0, (qval - level_high) / qval * left_border, right_border)
+    with warnings.catch_warnings():
```

Comment:
Could you fix it instead of using `warnings.catch_warnings()`?

Reply:
I could, for example like this:
```
qval_ = fns.where(qval == 0.0, fns.ones_like(qval), qval)
rb = fns.where(qval > 0.0, (qval - level_high) / qval_ * left_border, right_border)
```

But I decided just to intercept the warning on purpose. I wouldn't say that something is _wrong_ here to fix it, it's just the way numpy works processing both `where` branches even though only one of them will be used. 

Do you think it's a good idea to solve "cosmetic" problems by changing the computation logic?

Reply:
IMHO:This is more clear for me, because changes add 1 line of code and don't require additional comment in the code.

Reply:
Ok, I'll check whether there is difference in performance and switch to the code above if it's not significant

Reply:
With warning catching `tune_range` is about 7% slower, with a solution above it's about 20%. But still it's nothing compared to the rest of the pipeline. So I replace the warning catching with addition of one line.

Reply:
it looks like not good.  Maybe you can reuse ideas from here https://discuss.pytorch.org/t/torch-divide-only-where-denominator-is-non-zero/133968

Reply:
I've tried it out. For numpy this is indeed a preferred way. But for pytorch I had to do a workaround like this:
```
@numeric.divide.register(torch.Tensor)
def _(x1: torch.Tensor, x2: Union[torch.Tensor, float], where: Optional[torch.Tensor] = None) -> torch.Tensor:
    result = torch.clone(x1)
    if where is not None:
        result[where] = torch.divide(x1[where], x2[where])
    return result
```
This is far less effective, than it currently is. I propose to keep it as it is in the PR now.